src/Converter.ts

```diff
@@ -0,0 +1,142 @@
+const OpenAI = require("openai");
+require("dotenv").config();
+
+enum Framework {
```

Comment:
you can also move all the enums to separate files.
if we  would  like  to use the enums in other places  we would not have circular decency